envoy/network/connection.h

```diff
@@ -69,8 +69,9 @@ class ConnectionCallbacks {
 enum class ConnectionCloseType {
   FlushWrite, // Flush pending write data before raising ConnectionEvent::LocalClose
   NoFlush,    // Do not flush any pending data and immediately raise ConnectionEvent::LocalClose
```

Comment:
It would be better to mention that this type will give a try to write left data.

Reply:
addressed. thanks!